### DISCUSSION TITLE
8주차 스터디 회고

### DISCUSSION THREAD

**Discussion by alanhakhyeonsong** [Next-Level-Study/effective-java-study#24] [스터디 회고]
한 줄 회고와 건의사항을 말씀해주세요:)

**Comment by JeongJaew0n** [comment]
enum과 애노테이션의 다양한 사용법 알게 되어서 좋았습니다.
새해 복 많이 받으세요.

**Comment by sdh46** [comment]
처음으로 발표에 당첨됐는데 많이 떨렸고 미흡한 점들이 있어 좀 아쉬웠습니다. 
그리고 내일이면 새해인데 모두 새해 복 많이 받으세요

**Comment by SuJeong68** [comment]
2023년 마지막 스터디에 스스로가 해이해진 기분이 드네요. 2024년부터 다시 열심히 하겠습니다. 새해 복 많이 받으세요?:)

**Comment by alanhakhyeonsong** [comment]
요즘 업무가 바빠서 몇 주 참여 못하다 오랜만에 왔는데 다들 연말임에도 불구하고 참여하는 모습 보기 좋습니다.
새해에는 모두 성취하고 싶은 것들 모두 성취할 수 있는 한 해가 되길 바랍니다. (그전에 건강이 우선^^)
(저 아직 서른 아니에요^^;)

**Comment by narangd0** [comment]
어쩌다보니 2주간 잘 쉬게되었네요
내년부터 다시 본 스터디 목적 달성을 위해 같이 달려가봐영
같이 스터디 해줘서 감사하고 2024 새복많받!

**Comment by Kurmarkpzgr** [comment]
2주만이었는데, 다들 수고 많으셨습니다. 저는 다음주 주말 출근...이 예정되어 있어서 다음주 미참하게 되었습니다. 
새해 복 많이 받으세요
